.github/workflows/test.yml

```diff
@@ -11,6 +11,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - run: node --version
```

Comment:
This should not be needed, right?

Reply:
Whoops, that was there for debugging purposes. Good catch!
Removing.